whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -273,29 +291,20 @@ class ElasticSearch {
     }
 
     private static List<Isbn> getOtherIsbn(List<String> isbnValues) {
```

Comment:
Perhaps have this just take a String and use like `isbnValues.findResults { getOtherIsbn(it) }`?

Reply:
Yes, that's a bit cleaner. :gem: